src/AuthModule/components/Login/Login.tsx

```diff
@@ -1,11 +1,128 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { useForm } from "react-hook-form";
+import logo from "../../../assets/images/PMS 3.png";
+import { Link, useNavigate } from "react-router-dom";
+import { FormData } from "../../../interfaces/Auth";
+import { useState } from "react";
+import {useToast} from '../../../context/TostifyContext'
+import axios from "axios";
 
+export default function Login({savLoginData}) {
+  // All states here on the top 
+  const [showPassword, setShowPassword] = useState<boolean>(false);
```

Comment:
I suggest renaming this variable to `isPasswordShown` as naming it as  `showPassword`  will make it feel like it is a function, not a variable. By renaming it `isPasswordShown` this will make it easier guess that this is a variable and it contains a boolean value.